packages/core/notifications/core-notifications-browser-internal/src/toasts/__snapshots__/global_toast_list.test.tsx.snap

```diff
@@ -1,123 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Opted out of snapshot testing for `global_toast_list.tsx`, since it's test have been converted to RTL (so the tests focus on how the user would interact with the toast lists and the toasts that should be rendered) as opposed to testing the internals, that been said the snapshots generated with this approach don't offer much value since the test suites already asserts values that should be consistent in it's behaviour.